report_pre_thesis/src/sections/related_work.tex

```diff
@@ -1,14 +1,43 @@
-% Gaps in Current Research
-% - Identified shortcomings in existing work
-% Theoretical Frameworks
-% - Any theories or models that are foundational to the research
\ No newline at end of file
+Quantitative analysis of LIBS spectra is a well-studied problem.
+The LIBS community has developed a variety of techniques to analyze LIBS spectra in different context such as for soil and rock analysis \cite{huang_progress_2023}.
+To analyze the spectra a variety of both stastical and machine learning approaches can be used, and in this section we will present a variety of papers that present research that is related and relevant for our work.
+
+\citeauthor{takahashi_quantitative_2017} evaluated methods for correcting matrix effects and self-absorption in the quantitative analysis of solid compositions via LIBS.
+They found that while traditional calibration methods for LIBS face challenges with complex in-situ samples, CF-LIBS is limited by the need for prior sample knowledge.
+Multivariate analysis techniques like PCR and PLS handle matrix effects well but struggle with non-linearities like self-absorption.
+Despite limited research into the use of ANNs for quantitative analysis of solid samples, they found that ANN showed potential to overcome these issues because ANNs are able to learn the non-linear relationship between the LIBS spectra and the composition of the sample.
+However, ANNs is limited by the need for a large number of training samples.
+They conclude that simpler matrices favor theory-based methods, and more complex samples benefit from flexible statistical approaches, with computational advancements promising further improvements in LIBS analysis\cite{takahashi_quantitative_2017}.
+While this study applies ANNs to tackle LIBS data non-linearities, our research is oriented towards identifying which components in the MOC model most significantly influence the overall error. Although our methodologies differ, insights from the ANN approach could inform our series of experiments, potentially guiding us in identifying critical components in our model.
+
+\citeauthor{lepore_quantitative_2022} examine how dividing LIBS spectra into submodels, as presented in Anderson, impacts the prediction of major element compositions.
+Their findings highlight that, although submodeling is a promising approach, lower Root Mean Square Errors of Prediction (RMSE-P) are often obtained when using the entire spectrum, indicating that a comprehensive dataset generally leads to more accurate geochemical analysis\cite{lepore_quantitative_2022}.
+The insights from this study are informative for our project, as they offer useful considerations and approaches for identifying the components of the MOC model that most significantly impact the overall error.
+
+\citeauthor{bai_application_2023} explored elastic net regression for analyzing Mars-analog LIBS data and demonstrated that it efficiently balances feature selection with model stability.
```

Comment:
Question: what is Mars-analog?

If I don't know, it's likely readers won't either.